core/src/sigverify.rs

```diff
@@ -254,14 +237,10 @@ pub fn ed25519_verify(batches: &[SharedPackets]) -> Vec<Vec<u8>> {
     debug!("CUDA ECDSA for {}", batch_size(batches));
     let mut out = Vec::new();
     let mut elems = Vec::new();
-    let mut locks = Vec::new();
     let mut rvs = Vec::new();
 
-    for packets in batches {
-        locks.push(packets.read().unwrap());
-    }
     let mut num_packets = 0;
-    for p in locks {
+    for p in batches {
```

Comment:
@sakridge, this stuff didn't _need_ locks right? This change was made after your review.

Reply:
I don't believe so.. @aeyakovenko would know the best, he added the SharedPacket code. From this module's perspective it was just reading the packets.

Reply:
Yeah on second thought, this should be totally fine.